edocumentation-application/src/main/resources/application.yml

```diff
@@ -56,7 +56,7 @@ logging:
     root: ${ROOT_LOG_LEVEL:INFO}
     org:
       springframework: ${SPRING_LOG_LEVEL:INFO}
-      dcsa: ${DCSA_LOG_LEVEL:INFO}
+      dcsa: ${DCSA_LOG_LEVEL:DEBUG}
```

Comment:
Please keep this at `INFO` by default and instead tweak your run configuration to set `DCSA_LOG_LEVEL=debug` via environment variable.

Reply:
Oops, this was committed by mistake :-)